_overviews/core/custom-collections.md

```diff
@@ -337,7 +337,7 @@ That is it. The final [`Capped` class](#final-version-of-capped-class):
   `knownSize` operation is also overridden because the size is always
   known.
 
-Its implementation requires a little bit of protocol. In essence, you
+Its implementation requires a little of protocol. In essence, you
```

Comment:
"little of protocol" is wrong. I think the original is better. "requires adding protocol" would be okay too.